python/observability-online-eval/livekit/livekit-gemini.py

```
@@ -5,27 +5,53 @@
+from tavily import TavilyClient
 
 dotenv.load_dotenv(override=True)
-logging.basicConfig(level=logging.INFO)
+logging.basicConfig(level=logging.DEBUG)
 
 
 logger = Maxim().logger()
+TAVILY_API_KEY = os.getenv("TAVILY_API_KEY")
 
 
 def on_event(event:str, data:dict):
```

Ruff (0.11.9)

Missing return type annotation for public function `on_event`

Add return type annotation: `None`

(ANN201)